charts/argo-workflows/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.5.2
+version: 0.5.3
```

As this introduces backwards compatible functionality, lets bump the minor version :-)
```suggestion
version: 0.6.0
```